common/src/code_storage.rs

```diff
@@ -28,16 +28,66 @@ pub enum Error {
 
```

Comment:
frankly speaking, I disagree with such usage of our common storage traits, but let it be so atm

Reply:
We need to have more guarantees that if instrumented code exists in the storage, than the original one exists. This is reached by "default" trait's method implementation. The solution here is not final. Finalizing is on @gshep.